voter-dapp/README.md

```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
```

Comment:
Can we tear most of this documentation out? I think it's kinda overwhelming. IMO, it'd be nice to just leave the documentation for the commands we use in our normal dev flow:
- `npm start`
- `CI=true npm test`
- `npm run build`

And then a link to the react documentation website.

Reply:
Done.